library/agent/ServiceConfig.ts

```diff
@@ -1,29 +1,35 @@
 import { IPMatcher } from "../helpers/ip-matcher/IPMatcher";
 import { LimitedContext, matchEndpoints } from "../helpers/matchEndpoints";
 import { Endpoint } from "./Config";
-import { Blocklist as BlocklistType } from "./api/fetchBlockedLists";
+import { IPList } from "./api/fetchBlockedLists";
 
 export class ServiceConfig {
   private blockedUserIds: Map<string, string> = new Map();
-  private allowedIPAddresses: Map<string, string> = new Map();
+  private allowedIPAddresses: Set<string> = new Set();
```

Comment:
Lets comments what this is (bypass list, another feature)

Reply:
I see a comment has been added resolving this